src/js/update-sites.js

```diff
@@ -3,12 +3,14 @@
  *
  * @type {Element}
  */
-let link = document.querySelector('link[href="src/partials/listing.html"]');
-let content = link.import;
-let el = content.querySelector('script');
-document.querySelector('body').appendChild(el.cloneNode(true));
+let packageBaseURL = store.state.atlasEnvironments[store.state.env];
 
-let baseURL = store.state.atlasEnvironments[store.state.env];
+function remDup() {
+  $("#packages-to-add option").val(function(idx, val) {
```

Comment:
There should be no DOM manipulation in this application done by us. Vue nicely takes care of that, and so filtering the data before rendering it is the best option.